File: Service/Implement/ProductUnitService.cs
namespace Service.Implement
{
    public class ProductUnitService : BaseService<ProductUnit, IProductUnitRepository>
    , IProductUnitService
    {
    private readonly IProductUnitRepository _ProductUnitRepository;
    public ProductUnitService(IProductUnitRepository ProductUnitRepository) : base(ProductUnitRepository)
    {
    _ProductUnitRepository = ProductUnitRepository;
    }
    }
    }

File: CMS/src/app/shared/ProductUnit.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ProductUnit } from 'src/app/shared/ProductUnit.model';
import { BaseService } from './Base.service';
@Injectable({
  providedIn: 'root'
})
export class ProductUnitService extends BaseService {
  constructor(public httpClient: HttpClient) {
    super(httpClient);
    this.Controller = "ProductUnit";
  }
}

